db/fixtures/miq_product_features.yml

```diff
@@ -5123,3 +5123,36 @@
     :description: Delete Arbitration Settings
     :feature_type: admin
     :identifier: delete_arbitration_setting
+
+# Blueprints
+- :name: Blueprints
+  :description: All Blueprints Access Rules
+  :feature_type:
```

Comment:
@h-kataria thanks so much! Updated with your suggestions. Can you confirm :+1: / :-1: ?